src/lib/crypto/kimchi_bindings/stubs/dune

```diff
@@ -164,7 +163,7 @@
     CARGO_TARGET_DIR
     "%{read:dune-build-root}/cargo_kimchi_bindgen"
     (run cargo run %{targets}))
-   (run ocamlformat -i %{targets}))))
```

Comment:
We want to run `ocamlformat` here, otherwise the results are extremely ugly. The crypto team needs to explicitly look at these files.

Reply:
I see...

I assumed formatting was introduced mainly to avoid issues with committing files and running `make reformat-diff` before commit.